daemon/cmd/daemon_main.go

```diff
@@ -589,6 +589,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.String(option.LoadBalancerMode, option.NodePortModeSNAT, "BPF load balancing mode (\"snat\", \"dsr\", \"hybrid\")")
 	option.BindEnv(vp, option.LoadBalancerMode)
 
+	flags.String(option.LoadBalancerAlgConf, option.LoadBalancerAlgGlobal, "The source of load-balancing algorithm value for Service (\"global\", \"annotation\")")
```

Comment:
Would something like `bpf-lb-algorithm-annotation=true` make more sense? I assume that even in annotation mode, services without the annotation should fall back to the global option.

Reply:
yes that's right when `bpf-lb-algorithm-configuration=annotation` and the annotation is not present on the service then global configuration is applied. 
I personally preferred the original name instead of `bpf-lb-algorithm-annotation=true`

Reply:
Correct, services without the annotation fall back to the global option. In the end we only need this extra config given we make the BPF code more complex. I think `bpf-lb-algorithm-annotation=true` makes sense to me, @kl52752 wdyt?

Reply:
yea, makes sens, renamed